docs/pages/technical-specs/expo-updates-0.md

```diff
@@ -121,7 +121,7 @@ type Asset = {
   * `launchAsset`: A special asset that is the entry point of the application code. The `fileExtension` field will be ignored for this asset and SHOULD be omitted.
   * `assets`: An array of assets used by the update bundle, such as JavaScript, pictures, and fonts. All assets (including the `launchAsset`) should be downloaded to disk before executing the the update, and a mapping of asset `key`s to locations on disk should be provided to application code.
   * Properties of each asset object:
-    * `hash`: Case-insensitive, hex-encoded SHA-256 hash of the file to guarantee integrity.
+    * `hash`: Case-insensitive, base64-encoded SHA-256 hash of the file to guarantee integrity.
```

Comment:
```suggestion
    * `hash`: Case-insensitive, base64url-encoded SHA-256 hash of the file to guarantee integrity.
```
https://datatracker.ietf.org/doc/html/rfc4648#page-7

Reply:
this is how we do it server side: https://github.com/expo/universe/blob/main/server/www/src/utils/uris.ts#L13

Reply:
"Case-insensitive" should be removed.